src/control/control.cpp

```diff
@@ -102,7 +102,10 @@ QSharedPointer<ControlDoublePrivate> ControlDoublePrivate::getControl(
         bool bTrack,
         bool bPersist,
         double defaultValue) {
-    VERIFY_OR_DEBUG_ASSERT(key.isValid()) {
+    if (!key.isValid()) {
```

Comment:
This code removes the debug assertion in case of an invalid key in all cases.
I think that is not required to fix the crash.

Reply:
Huh? This `if` statement executes if the key is invalid.

Reply:
Ah, I see. Than the condition inside the DEBUG_ASSERT below is redundant.

Reply:
Yes, it is, but it is still useful to trigger an assertion conditionally depending on the ControlFlag.